app/models/taxonomy.rb

```diff
@@ -1,4 +1,10 @@
 class Taxonomy < ApplicationRecord
+  apipie :prop_group, name: :taxonomy_props do
+    property :description, String, desc: "Description of the #{@meta[:class_scope]}"
+    property :created_at, String, desc: "The time when the #{@meta[:class_scope]} was created"
+    property :updated_at, String, desc: "The last time when the #{@meta[:class_scope]} was updated"
+  end
```

Comment:
just curious, why did you decide to split these three from the title which is defined in the subclasses?

Reply:
well... It's just because I've got this before other tweaks and somehow I forgot to remove it :facepalm: 

Updated, thanks :)